packages/generator-volto/generators/addon/templates/Makefile.tpl

```diff
@@ -0,0 +1,54 @@
+SHELL=/bin/bash
+
+DIR=$(shell basename $$(pwd))
+ADDON ?= "<%= addonName %>"
+
+# We like colors
+# From: https://coderwall.com/p/izxssa/colored-makefile-for-golang-projects
+RED=`tput setaf 1`
+GREEN=`tput setaf 2`
+RESET=`tput sgr0`
+YELLOW=`tput setaf 3`
+
+project:
+	npm install -g yo
+	npm install -g @plone/generator-volto
+	npm install -g mrs-developer
```

Comment:
Globally?

Reply:
It's based on Volto docs :) https://docs.voltocms.com/getting-started/install/#install-volto

Reply:
I always install these globally 😄